src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -85,6 +86,16 @@ private static void getLocationsFromPath(List<String> possiblePaths) {
         if (path.endsWith("google-cloud-sdk" + File.separator + "bin")) {
           possiblePaths.add(path.substring(0, path.length() - 4));
         }
+
+        Path possibleLink = Paths.get(path, "gcloud");
+        if (Files.exists(possibleLink) && Files.isSymbolicLink(possibleLink)) {
```

Comment:
On my Linux, the `gcloud` executable is not a symlink. (I just unzipped the SDK zip.) So I think `isSymbolicLink()` is always false for me.